compiler/src/dotty/tools/dotc/coverage/Serializer.scala

```diff
@@ -32,7 +32,7 @@ object Serializer:
 
     def getRelativePath(filePath: Path): String =
       val relPath = sourceRoot.relativize(filePath)
-      relPath.toString
+      relPath.toString.replace("\\", "/") // stick to unix path format
```

Comment:
So I think this may break deserialization if we do this. For example it's up to scoverage to read this up, and it doesn't replace it: https://github.com/scoverage/scalac-scoverage-plugin/blob/V2/serializer/src/main/scala/scoverage/serialize/Serializer.scala#L121 so I think this would fail here.

Sort of related to all this @TheElectronWill you mentioned in the docs that were merged that users can use the milestone release of sbt-scoverage, but sbt-scoverage still has an old version hardcoded here:

https://github.com/scoverage/sbt-scoverage/blob/73f8f92e41eddc2dad3a7404284b5aa47afaee52/src/main/scala/scoverage/ScoverageSbtPlugin.scala#L162

So I don't believe this will actually work. Now that this is merged in it'd be a good idea to get some tests in sbt-scoverage to do this.

As a total aside just to be transparent, while I have admin access and am happy to help anyone that would like to contribute to scoverage, I'm not actively working on it. So again, this is amazing that this is merged in, huge kudos @TheElectronWill! There is however some more work that will probably need to be done on the scoverage side to ensure everything is usable for people.

Reply:
Maybe, we can just skip those tests on windows?  

I am not sure if `File(...)` is able to handle Linux style paths on windows, I think that `Paths.get(...)` is.

Reply:
According to this [SO question](https://stackoverflow.com/questions/2417485/difference-between-file-separator-and-slash-in-paths), `/` works everywhere, including on windows :)